packages/react/src/combobox/ComboboxBase/ComboboxMultiBase.tsx

```diff
@@ -103,6 +106,18 @@ export function ComboboxMultiBase<Option extends DefaultComboboxOption>({
 }: ComboboxMultiBaseProps<Option>) {
 	const inputRef = useRef<HTMLInputElement>(null);
 
+	if (process.env.NODE_ENV !== 'production' && maxWidthProp) {
+		validateMaxWidth('ComboboxMulti', maxWidthProp);
+
+		if (block) {
+			// Future enhancement: Automatically set block to false if maxWidth is defined.
```

Comment:
Should this be a ticket rather than a comment?

Reply:
Removed because it's actually a breaking change